library/src/main/java/in/krharsh17/barview/BarGroup.java

```diff
@@ -295,6 +364,9 @@ else if (s.length() >= 8 && s.length() < 12) {
         return finalizedString.toString();
     }
 
+    /**
+     * ?
+     */
```

Comment:
@parthsharma07 Please provide the documentation for this

Reply:
please do reply @parthsharma07 :)